tests/execution/call-mutation.md

```diff
@@ -20,14 +20,14 @@ input PostInputWithoutUserId {
 
 type Mutation {
   attachPostToFirstUser(postId: Int!): User
-    @call(mutation: "attachPostToUser", args: {postId: "{{args.postId}}", userId: 1})
+    @call(steps: [{mutation: "attachPostToUser"}, {args: {postId: "{{args.postId}}", userId: 1}}])
```

Comment:
How his this even working? Shouldn't it be as follows? If not we should throw a validation error.
```suggestion
    @call(steps: [{mutation: "attachPostToUser", args: {postId: "{{args.postId}}", userId: 1}}])
```

Reply:
it's vector of steps, here it's working like an enum of type mutation arg or query

Reply:
oh got your point.. it's working because I am extending the args